packages/calendar/test/calendar.test.ts

```diff
@@ -10,23 +10,67 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
-import sinon, { spy } from 'sinon';
-
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
 import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
-import { Calendar } from '@spectrum-web-components/calendar';
-
+import { spy } from 'sinon';
```

Comment:
I found this neccessary for checking the component's change event. In SWC it seems like sinon is used a lot, even in those testing-helpers, but here I need just the spy functionality.

Related to: https://github.com/adobe/spectrum-web-components/pull/3856#discussion_r1705118930